doc/release-notes/6432-basic-oidc-support.md

```diff
@@ -0,0 +1,16 @@
+# Basic OpenID Connect support
+Working on epic #5974 brought us refactored code for our OAuth 2.0 based login options.
+This has now been extended to provide basic support for any OpenID Connect compliant
+authentication provider.
+
+While with our OAuth 2.0 login options you had to implement support for every provider
+by pull request, OpenID Connect provides a standardized way for authentication, user 
+details and more. You are able to use any provider just by loading a configuration file,
+without touching the codebase.
```

Comment:
Does this mean that #4334 could be addressed without a pull request? Previously pull request #4341 was made to add support for a country-specific OAuth provider but it was closed as "we want this to be a plugin outside the core code". My understanding is that @philippconzett and @oodu run a fork of Dataverse because they need support for their country-specific login. It would be great if they could stop running a fork and get the auth support they need with configuration!

Reply:
It looks like this will be the case.
https://docs.feide.no/service_providers/integration_guide/oauth_oidc/openidconnect.html

Reply:
Thanks, looks like Obi has commented on the issue and can repopen if needed.